plugin/src/main/scala/com/geirsson/CiReleasePlugin.scala

```diff
@@ -22,6 +22,14 @@ import xerial.sbt.Sonatype.autoImport._
 
 object CiReleasePlugin extends AutoPlugin {
 
+  object autoImport {
+    val cireleasePublishStableRelease =
```

Comment:
Thanks for the contribution!

I find the idea of "stable version" to be confusing because the word can mean different things in different context.
For example, when people look at a version like `3.0.1-RC1-bin-20210413-f3c1468-NIGHTLY`, they'd go "yea. that's a stable version".

Instead, I'd propose a setting like `cireleaseRequireTag` (default: true), and when set to `false`, run the `+publishSigned` etc. That way, it would do what you want I think, and we don't have to name that nightly version thing.